### PR TITLE
Move manpages to scdoc

### PR DESCRIPTION
**Pull request by chrisduerr**
This rewrites the existing manpages to use the `scdoc` format, making it simpler to read and edit the manpages without intricate roff knowledge.

Some minor changes to the manpages were made in the process, mostly focusing on correcting some of the wording. The list of maintainers has also changed to ensure people not involved in the project anymore aren't unnecessarily contacted for support.

### CODE REVIEW

File: extra/man/alacritty.scd [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+	Command and args to execute (must be last argument)
+
+*--config-file* <config-file>
+	Specify alternative configuration file
+
+	Alacritty looks for the configuration file at the following paths:
+		. $XDG_CONFIG_HOME/alacritty/alacritty.yml
+		. $XDG_CONFIG_HOME/alacritty.yml
+		. $HOME/.config/alacritty/alacritty.yml
+		. $HOME/.alacritty.yml
```
Comment (kchibisov) [inline extra/man/alacritty.scd:56]:
```suggestion
		- $XDG_CONFIG_HOME/alacritty/alacritty.yml
		- $XDG_CONFIG_HOME/alacritty.yml
		- $HOME/.config/alacritty/alacritty.yml
		- $HOME/.alacritty.yml
```

File: extra/man/alacritty.scd [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+	Prints version information
+
+# OPTIONS
+
+*--class* <general> | <general>,<instance>
+	Defines the window class hint on Linux [default: Alacritty,Alacritty]
+
+	When only the general class is passed, instance will be set to the same value.
+
+	On Wayland the general class sets the `app_id`, while the instance class is ignored.
```
Comment (kchibisov) [inline extra/man/alacritty.scd:44]:
```suggestion
	On Wayland the general class sets the _app\_id_, while the instance class is ignored.
```

File: extra/man/alacritty-msg.scd [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+		*OPTIONS*
+			*--hold*
+				Remain open after child process exits
+
+			*--working-directory* <working-directory>
+				Start the shell in the specified working directory
+
+			*command* <command>...
+				Command and args to execute (must be last argument)
```
Comment (kchibisov) [inline extra/man/alacritty-msg.scd:30]: I feel like it looks better without extra `\t`.

**Review by kchibisov — CHANGES_REQUESTED** [review]
Could you name the files with the `1` in their name? So once we have the config file we'd properly have it in `5` with the same `alacritty` name?

I'd also update editor config to make `tab` to be `4` spaces in `scd` files, because it's to what it expands in the end.

File: extra/man/alacritty-msg.scd [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+	*ARGS*
+		*<CONFIG_OPTIONS>...*
+			Configuration file options [example: cursor.style=Beam]
+
+	*OPTIONS*
+		*-w, --window-id* <WINDOW_ID>
+			Window ID for the new config.
+
+			Use `-1` to apply this change to all windows.
```
Comment (kchibisov) [inline extra/man/alacritty-msg.scd:43]:
```suggestion
			Use _-1_ to apply this change to all windows.
```

File: extra/man/alacritty.scd (hunk 1 above)
Comment (chrisduerr) [reply extra/man/alacritty.scd:56]: Why do you suggest this change? This was an ordered list before so this is intentional.

File: extra/man/alacritty-msg.scd (hunk 3 above)
Comment (chrisduerr) [reply extra/man/alacritty-msg.scd:30]: Which part?

File: extra/man/alacritty.scd (hunk 1 above)
Comment (kchibisov) [reply extra/man/alacritty.scd]: I simply suggest to use `-` to form a character you want anyway. Nothing will be changed other than using bigger `dot`.

File: extra/man/alacritty.1.scd [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+*--config-file* <config-file>
+	Specify alternative configuration file
+
+	Alacritty looks for the configuration file at the following paths:
+		. $XDG_CONFIG_HOME/alacritty/alacritty.yml
+		. $XDG_CONFIG_HOME/alacritty.yml
+		. $HOME/.config/alacritty/alacritty.yml
+		. $HOME/.alacritty.yml
+
+	On Windows, the configuration file is located at %APPDATA%\alacritty\alacritty.yml.
```
Comment (kchibisov) [inline extra/man/alacritty.1.scd:58]:
```suggestion
	On Windows, the configuration file is located at %APPDATA%\\alacritty\\alacritty.yml.
```

File: extra/man/alacritty.scd (hunk 1 above)
Comment (kchibisov) [reply extra/man/alacritty.scd]: Haven't seen the numbers generated.

File: extra/man/alacritty-msg.scd (hunk 3 above)
Comment (kchibisov) [reply extra/man/alacritty-msg.scd]: Like all I've selected, so `OPTION` is not shifted.

File: .editorconfig [hunk 6]
```diff
@@ -16,3 +16,6 @@ indent_size = 2
 
 [Makefile]
 indent_style = tab
+
+[*.sdc]
```
Comment (kchibisov) [inline .editorconfig:20]:
```suggestion
[*.scd]
```

And also a `tab_width=4`.

File: extra/man/alacritty.1.scd [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+	Alacritty looks for the configuration file at the following paths:
+		. $XDG_CONFIG_HOME/alacritty/alacritty.yml
+		. $XDG_CONFIG_HOME/alacritty.yml
+		. $HOME/.config/alacritty/alacritty.yml
+		. $HOME/.alacritty.yml
+
+	On Windows, the configuration file is located at %APPDATA%\\alacritty\\alacritty.yml.
+
+*--embed* <parent>
+	X11 window ID to embed Alacritty within (decimal or hexadecimal with "0x" prefix)
```
Comment (kchibisov) [inline extra/man/alacritty.1.scd:61]: Maybe like that will look better?

```suggestion
	X11 window ID to embed Alacritty within (decimal or hexadecimal with _0x_ prefix)
```

File: extra/man/alacritty-msg.scd (hunk 3 above)
Comment (kchibisov) [reply extra/man/alacritty-msg.scd]: You can look at how `config` and its `OPTION` looks, it's indented correctly.

**Review by kchibisov — APPROVED** [review]

**Pull request merged**